eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -339,6 +339,16 @@ public void testFindByRegionsAndComplexFiltersExcludeSingleRoot() {
         }
     }
 
+    @Test
+    public void testFindDistinctChromosomesByStudyId() {
+        List<String> chromosomeList = variantEntityRepository.findDistinctChromosomes();
+
+        List<String> expectedChromosomeList = new ArrayList<>();
+        expectedChromosomeList.add("11");
+
+        assertEquals(expectedChromosomeList, chromosomeList);
```

Comment:
expectedChromsomeList can be declared using `Collections.singletonList("11")`, inside the assertEquals (or outside if you prefer for readibility).

This is an small method and is not really necessary here, but in bigger test methods reducing the number of lines improves the readibility.